impl/client/src/main/javascript/web/pentaho/visual/samples/calc/model.js

```diff
@@ -57,9 +57,9 @@ define([
           {
             name: "operation",
             type: {
-              base: "refinement",
-              of:   "string",
-              facets: ["DiscreteDomain"],
+              base: "string",
```

Comment:
When using specifications, the default value type of a **root** property is "complex", so here, we need to specify "number" to override the default. On inherited properties, the default value type is the base property's type.
In the other models, like in the `.../models/types/colorSet`? In these the base type, `"string"` is written in code: `PentahoString.extend( ... );`